ieeeuoft/hackathon-template
# Ieee 146 only count healthy items towards quantity remaining

## Overview

- Resolves #IEEE-146
- Removed order items with part returned health of "Healthy" from checked out items in the annotation


## Unit Tests Created

- added a unit test to verify items returned with status other than Healthy or not counted to quantity remaining


## Steps to QA

- Create an order w/ order items on admin dashboard
- mark that order as picked up
- call the hardware api to see the quantity_remaining field change (decrease by however many you checked out) for a hardware item
- mark the order item has Healthy
- call hardware api again, quantity_remaining should've increased by 1
